libs/components/code-examples/src/lib/modules/forms/checkbox/help-key/example.component.html

```diff
@@ -1,33 +1,31 @@
-<div class="sky-padding-even-xl">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.